maintenance_equipment_scrap/models/maintenance_equipment.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class MaintenanceEquipment(models.Model):
+
+    _inherit = 'maintenance.equipment'
+
+    equipment_scrap_template_id = fields.Many2one(
+        'mail.template',
+        string='Equipment Scrap Email Template',
+        default=(lambda self:
+                 self.env.user.company_id.equipment_scrap_template_id)
+    )
+
+    @api.multi
+    def action_perform_scrap(self):
+        self.ensure_one()
+        action = self.env.ref(
+            'maintenance_equipment_scrap.wizard_perform_equipment_scrap_action'
+        )
+        result = action.read()[0]
+        return result
```

Comment:
OK, thanks for the explanation. Indeed it is a good step forward.
I'm pretty sure we also need a Stage/State, but I agree that it could be left for a later PR.
In case you are willing to add that, I strongly suggest reusing the oca/server-tools base_stage_state module.